arch/inst/I/fence.tso.yaml

```diff
@@ -0,0 +1,42 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: fence
+long_name: Memory ordering fence
```

Comment:
(Looking for a different long_name than for FENCE...)
"Memory ordering fence, total store ordering"?